internal/app/config.go

```diff
@@ -184,6 +184,14 @@ func (a *App) envVarNatsOverrides() error {
 		return errors.New("missing parameter: nats.url")
 	}
 
+	if a.v.GetString("nats.publisherSubjectPrefix") != "" {
```

Comment:
Definitely not a concern now, but I think the PublisherSubjectPrefix is something that should be coded and not configured.